support/cas-server-support-otp-mfa-core/src/main/java/org/apereo/cas/otp/web/flow/OneTimeTokenAccountCheckRegistrationAction.java

```diff
@@ -45,9 +48,12 @@ protected Event doExecute(final RequestContext requestContext) {
         if (acct == null || StringUtils.isBlank(acct.getSecretKey())) {
             val keyAccount = this.repository.create(uid);
             val keyUri = "otpauth://totp/" + this.label + ':' + uid + "?secret=" + keyAccount.getSecretKey() + "&issuer=" + this.issuer;
+            ByteArrayOutputStream QRcodeBase64 = new ByteArrayOutputStream();
+            QRUtils.generateQRCode(QRcodeBase64, keyUri, QRUtils.WIDTH_LARGE, QRUtils.WIDTH_LARGE);
+
             val flowScope = requestContext.getFlowScope();
             flowScope.put(FLOW_SCOPE_ATTR_ACCOUNT, keyAccount);
-            flowScope.put(FLOW_SCOPE_ATTR_ACCOUNT_URI, keyUri);
+            flowScope.put(FLOW_SCOPE_ATTR_QR_IMAGE_BASE64, QRcodeBase64);
```

Comment:
Renaming a flow-scope variable will surely break tests. Please review and adjust.

Reply:
Right, I've fixed the tests as well. We should be okay now